.planemo.sh

```diff
@@ -9,9 +9,10 @@ echo "BLA"
 # source deactivate
 planemo database_create galaxy
 curl https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -o miniconda.sh
+chmod +x miniconda.sh
 ./miniconda.sh -b -p $tmp_dir/conda
-planemo conda_init --conda_prefix $tmp_dir/conda
-export PATH=$tmp_dir/conda/bin:$PATH
+# planemo conda_init --conda_prefix $tmp_dir/conda
```

Comment:
planemo init should work:
https://github.com/galaxyproject/tools-iuc/blob/master/.travis.yml#L21

and install_galaxy should also not be necessary ... what problems are you facing?